terraform/modules/application/parameters.tf

```diff
@@ -3,7 +3,7 @@ data "aws_ssm_parameter" "partial_database_connection_string" {
 }
 
 resource "aws_ssm_parameter" "complete_database_connection_string" {
-  #checkov:skip=CKV_AWS_337:default encryption not using a kms cmk sufficient
+  key_id = aws_kms_key.this.arn

```

Comment:
Can we keep name at the top here? (I think this is coming from required vs non-required parameters, or what's more immediately going to want to be known to a human reading this - or maybe just that standard terraform examples always start with name?)

Reply:
Will update this here and a few other places where we have a similar thing 👍

Reply:
Done 👍